data_layer/sqitch/deploy/labellisation/export.sql

```diff
@@ -0,0 +1,105 @@
+-- Deploy tet:labellisation/export to pg
+
+BEGIN;
+
+
+create view labellisation.export_score_audit as
+with
+    score_audit as (
+        with
+            last_audit as (
+                with last_audit_date as (
+                    -- Récupérer le dernier audit par collectivité
+                    select collectivite_id, referentiel, max(date_debut) as date_debut
+                    from labellisation.audit
+                    where date_debut is not null
+                    group by collectivite_id, referentiel
+                )
+                select a.*
+                from labellisation.audit a
+                         join last_audit_date l
```

Comment:
un `order by date_debut desc where date_debut is not null limit 1` ne serait-il pas plus simple qu'une CTE ?

Reply:
Je veux le dernier audit par collectivité et référentiel